src/components/ui/Navigation.vue

```diff
@@ -16,7 +16,7 @@
           <router-link to="/">Support</router-link>
         </div>
         <div class="navbar-item connect">
-          <a href="" v-on:click="setAddress">Connect</a>
+          <a href="" v-on:click="setPublicWalletAddress">Connect</a>
```

Comment:
PublicWalletAddress is some new word. i hate new words. let's decide what to call the public key and use it everywhere. 
* Wallet is overloaded and meaningless
* Address does suck
* PublicKey is nice...

Reply:
Think we should stick syntactically consistent with using the shorthand for all `v-on:`'s and just use `@`'s instead.

Reply:
changing them all to publicKey